internal/cli/opsmanager/logs/decrypt.go

```diff
@@ -42,12 +44,23 @@ func (opts *DecryptOpts) initFiles() func() error {
 	}
 }
 
+func (opts *DecryptOpts) stdOutMode() bool {
+	return opts.Out == ""
+}
+
 func (opts *DecryptOpts) Run() error {
-	outWriter, err := opts.NewWriteCloser()
-	if err != nil {
-		return err
+	var outWriter io.WriteCloser
+
+	if opts.stdOutMode() {
```

Comment:
I think `/dev/stdout` is not cross platform, have we tested it on windows?

Reply:
in the os package I see Stdout used one line below defined like this: `Stdout = NewFile(uintptr(syscall.Stdout), "/dev/stdout")`
so it might be further in the framework where /dev/stdout is mapped to the correct os stdout.
Will try in a windows VM

Reply:
in windows, for os.OpenFile("/dev/stdout",...) I got "open /dev/stdout: The system cannot find the path specified." 
while os.Stdout seems to work.
I guess we can leave the current implementation here and revisit the usage of /dev/stdout in the other places

Reply:
@tibulca would you open a Jira for a follow up? If we have code that is broken on Windows we have to fix it.